docs/Common/Enums/index.md

```diff
@@ -925,6 +941,54 @@ Radio Buttons
 |ON|0||
 |OFF|1||
 
+### LightName
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|FRONT_LEFT_HIGH_BEAM|0||
+|FRONT_RIGHT_HIGH_BEAM|1||
+|FRONT_LEFT_LOW_BEAM|2||
+|FRONT_RIGHT_LOW_BEAM|3||
+|FRONT_LEFT_PARKING_LIGHT|4||
+|FRONT_RIGHT_PARKING_LIGHT|5||
+|FRONT_LEFT_FOG_LIGHT|6||
+|FRONT_RIGHT_FOG_LIGHT|7||
+|FRONT_LEFT_DAYTIME_RUNNING_LIGHT|8||
+|FRONT_RIGHT_DAYTIME_RUNNING_LIGHT|9||
+|FRONT_LEFT_TURN_LIGHT|10||
+|FRONT_RIGHT_TURN_LIGHT|11||
+|REAR_LEFT_FOG_LIGHT|12||
+|REAR_RIGHT_FOG_LIGHT|13||
+|REAR_LEFT_TAIL_LIGHT|14||
+|REAR_RIGHT_TAIL_LIGHT|15||
+|REAR_LEFT_BRAKE_LIGHT|16||
+|REAR_RIGHT_BREAK_LIGHT|17||
```

Comment:
`REAR_RIGHT_BREAK_LIGHT` needs to be changed as well

Reply:
@jacobkeeler, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/103/commits/e7d66dee1aea44accd4d2caedf6dbf659a6bfb0b